Dockerfile

```diff
@@ -39,7 +39,8 @@ RUN wget -nv https://github.com/gophish/gophish/releases/download/${GOPHISH_VERS
 unzip gophish-v${GOPHISH_VERSION}-linux-64bit.zip && \
 rm -f gophish-v${GOPHISH_VERSION}-linux-64bit.zip
 
-RUN chmod +x gophish && ln -snf /run/secrets/config.json config.json
+RUN chmod +x gophish && ln -snf /run/secrets/config.json config.json && \
```

Comment:
Won't this fail if something has been mounted at `./data`?  I think a separate line like this would be better:
```
RUN [[ -e data ]] || mkdir data
```

Reply:
Actually, maybe it is OK as-is.

Reply:
Related but not in PR.  You can save a layer in the container by simplifying this.  If the script in the repo has the executable set, it will be copied that way into the container. 
https://github.com/cisagov/gophish-docker/blob/edd97bff9f9bf3947dee1a5bc36187abfb149c84/Dockerfile#L33-L34
So... make the `get-api-key` executable, and remove the `chmod` line from the Dockerfile.

Reply:
You are correct; I think I have a simpler solution: `mkdir -p data`

```$ mkdir --help
Usage: mkdir [OPTION]... DIRECTORY...
Create the DIRECTORY(ies), if they do not already exist.

Mandatory arguments to long options are mandatory for short options too.
  -m, --mode=MODE   set file mode (as in chmod), not a=rwx - umask
  -p, --parents     no error if existing, make parent directories as needed
...
```

Reply:
I still think you're better off changing the config to point to something like `/var/lib/gophish/gophish.db` and later bind-mounting that directory at runtime if persistence is desired.  But if this works for you then OK.

Reply:
Fixed in https://github.com/cisagov/gophish-docker/pull/5/commits/d27bc0f3e0fe53712ab980c8b8bf90f934205ed1

Reply:
@jsf9k take a look at the NOTE that I added to the initial comment above.  The config won't let us change the location of `gophish.db`.  I could use `/var/lib/gophish/gophish.db` instead of `/home/cisa/data`, but I'd still have to use a symlink to point to where the file resides.